src/Microsoft.Extensions.FileProviders.Abstractions/NullFileProvider/NullFileInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
@davidfowl I think the layering here is a little strange. Since most of the helper classes already exist in the `Sources` project which we do not publish but should reuse, we should just create another project that contains the NullFileProvider which reference Sources like the other FileProviders.